week-1/errors/4.js

```diff
@@ -1,2 +1,2 @@
-const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+const HourClockTime = "20:53";
```

Comment:
this is not a great variable name.

Reply:
@jcholyhead  Is it because it started with a capital letter?
should it be const hourClockTime?